gen/python/sym/__init__.py

```diff
@@ -1,21 +1,8 @@
 # -----------------------------------------------------------------------------
 # This file was autogenerated by symforce from template:
-#     geo_package/__init__.py.jinja
+#     function/namespace_init.py.jinja
 # Do NOT modify by hand.
 # -----------------------------------------------------------------------------
 
-"""
-Python runtime geometry package.
-"""
```

Comment:
We definitely still want the `sym` module docstring to show up properly, should make sure this doesn't break that

Reply:
Oops. Forgot about this. I made sure to check that it was there when initially when I was using the more elaborate `__init__.py` that allowed re-exporting, but forgot to do that here.

Reply:
Okay. Added it in